updating links (#210)
rstudio/education.rstudio.com: content/teach/how-to-teach/index.md

While it might seem like teaching data science is a new activity, educational professionals have studied teaching techniques for decades. Today's teaching techniques are based not only on the personal experiences of classroom teachers, but also on neuroscience and cognitive psychology. While we can't replicate the thoroughness of the National Academies's [_How People Learn_](https://www.amazon.com/How-People-Learn-II-Learners/dp/0309459648/), we can recommend a couple of starting points for learning how to teach technology:

> - **Read [_Teaching Tech Together_](http://teachtogether.tech).** RStudio's [Greg Wilson](/author/greg) penned (actually, he typed it) this book based on his work for [the Carpentries](http://carpentries.org), and it will help you create and deliver lessons that work for technical and non-technical audiences alike and to build a teaching community around them. Greg also created a free online webinar titled [*What Every Data Scientist Should Know About Education*](https://resources.rstudio.com/webinars/what-every-data-scientist-should-know-about-education). This one-hour introduction presents a handful of evidence-based practices you can use right now, explains why we believe they are true, and points you at other resources that will help you go further.

Reviewer: This link doesn't go to the webinar. If there isn't a working link, we can take out this reference.

Author: updated.

> - **Read [_Teaching Tech Together_](http://teachtogether.tech).** RStudio's [Greg Wilson](/author/greg) penned (actually, he typed it) this book based on his work for [the Carpentries](http://carpentries.org), and it will help you create and deliver lessons that work for technical and non-technical audiences alike and to build a teaching community around them. Greg also created a free online webinar titled [*What Every Data Scientist Should Know About Education*](https://cloud.rstudio.com/resources/webinars/what-every-data-scientist-should-know-about-education/). This one-hour introduction presents a handful of evidence-based practices you can use right now, explains why we believe they are true, and points you at other resources that will help you go further.

Reviewer: @RhoBott This link works but there's a wonky double banner on top, do you know if this is the best link we have for these old webinars?

Author: Good q. I couldn't find anything better, checking in with the web team to see if they have a good answer _or_ if there's an oppty to do _something_ w that goofy double banner. Will keep you posted here.

Author: @mine-cetinkaya-rundel Update: as far as I can tell, that's the only link. I think our options are to run the link w/ the goofy 2x banner or take the resource out entirely; lmk which you'd prefer .

Reviewer: We can keep it. I suggested an edit below.

> - **Read [_Teaching Tech Together_](http://teachtogether.tech).** RStudio's [Greg Wilson](/author/greg) penned (actually, he typed it) this book based on his work for [the Carpentries](http://carpentries.org), and it will help you create and deliver lessons that work for technical and non-technical audiences alike and to build a teaching community around them. Greg also created a free online webinar titled [*What Every Data Scientist Should Know About Education*](https://cloud.rstudio.com/resources/webinars/what-every-data-scientist-should-know-about-education/). This one-hour introduction presents a handful of evidence-based practices you can use right now, explains why we believe they are true, and points you at other resources that will help you go further.

Reviewer: ```suggestion
- **Read [_Teaching Tech Together_](http://teachtogether.tech).** Authored by [Greg Wilson](/author/greg) based on his work for [the Carpentries](http://carpentries.org), and this book will help you create and deliver lessons that work for technical and non-technical audiences alike and to build a teaching community around them. Greg also created a free online webinar titled [*What Every Data Scientist Should Know About Education*](https://cloud.rstudio.com/resources/webinars/what-every-data-scientist-should-know-about-education/). This one-hour introduction presents a handful of evidence-based practices you can use right now, explains why we believe they are true, and points you to other resources that will help you go further.
```

- **Get involved in [the Carpentries](http://carpentries.org)**, a volunteer organization that teaches foundational coding and data science skills to researchers worldwide (they've even run a workshop in Antarctica). Their lessons on [data wrangling](https://datacarpentry.org/lessons/), [programming](https://software-carpentry.org/lessons/), and [for librarians](https://librarycarpentry.org/lessons/) are all open access and classroom-tested.

> - **Become a [certified trainer](https://rstd.io/trainers).** In 2019, Posit launched an instructor training and certification program that aims to help people apply modern evidence-based teaching practices to teach data science using R and Posit’s products, and to help people who need such training find the trainers they need. You can meet our Certified Trainers and find out more about the program and the application process [here](https://rstd.io/trainers), and access our Instructor Training Materials [here](https://drive.google.com/drive/folders/13ohFt3D0EJ5PDbMaWTxnHH-hwA7G0IvY).

Reviewer: The materials link also seems to be dead, let's take it out.

Author: rewritten!

> - **Connect with the #rstats education community.** The [Posit Community](https://forum.posit.co/) is a warm and welcoming online discussion forum to ask (and answer!) any questions about using R. But you may not know that the site has a dedicated category for all things related to [teaching](https://forum.posit.co/). There, you'll find lively discussions between first-time instructors and seasoned educators about topics like how to introduce functions in R, ideas for datasets to use, or how to pronounce an R package name. To meet other R educators in person, we host a yearly [`rstudio::conf`](https://rstd.io/conf) that always features talks about teaching; you can browse past videos of talks in this track [here](https://resources.rstudio.com/teaching). If you do attend our conference, keep an eye on [Community](https://forum.posit.co/) before you go for announcements about the [_Birds of a Feather_ meetup](https://community.rstudio.com/tags/c/rstudio-conf/bof) for educators &mdash; this is an informal get together of like-minded R educators to talk shop and connect in person while you're all in the same place at the same time.

Reviewer: Is there an equivalent/working Posit Forum link we can use?

Author: added tags for teaching/education, updated conf resources
